docs/src/main/asciidoc/spanner.adoc

```diff
@@ -963,6 +964,17 @@ public String doWorkInsideTransaction() {
 
 When creating custom repositories for your own domain types and query methods, you can extend `SpannerRepository` to access Cloud Spanner-specific features as well as all features from `PagingAndSortingRepository` and `CrudRepository`.
 
+=== Query Methods
+
+`SpannerRepository` supports Query Methods.
+Described in the following sections, these are methods residing in your custom repository interfaces whose implementations are generated based on their names and annotations.
+Query Methods can read, write, and delete entities in Cloud Spanner.
+Parameters to these methods can be any Cloud Spanner data type supported directly or via custom configured converters.
+Parameters can also be of type `Struct` or POJOs.
+If a POJO is given as a parameter, it will be converted to a Struct with the same type-conversion logic as used to create write mutations.
```

Comment:
```suggestion
If a POJO is given as a parameter, it will be converted to a `Struct` with the same type-conversion logic as used to create write mutations.
```